x/rollapp/keeper/grpc_query_validate_genesis_bridge.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
+)
+
+func (k Keeper) ValidateGenesisBridge(goCtx context.Context, req *types.QueryValidateGenesisBridgeRequest) (*types.QueryValidateGenesisBridgeResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	ra, ok := k.GetRollapp(ctx, req.GetRollappId())
+	if !ok {
+		return nil, status.Error(codes.InvalidArgument, types.ErrRollappNotFound.Error())
+	}
+
+	// it's okay to use this value for simulation
+	channelID := "channel-0"
+	if ra.ChannelId != "" {
+		channelID = ra.ChannelId
+	}
+
+	_, err := types.NewGenesisBridgeValidator(req.Data, ra.GenesisInfo, channelID, ra.RollappId).Validate()
```

Comment:
if `Validate()` will be restructured, `channelID` is not required for validation